src/command.js

```diff
@@ -209,16 +216,48 @@ export default class StorybookCommand extends Command {
           exclude: serializeRegExp(parameters?.percy?.exclude),
           id
         }));
-      });
+      }, WAIT_FOR_TIMEOUT);
     } catch (error) {
       // remove stack traces from any eval error
       if (typeof error === 'string') {
-        throw new Error(error.replace(/^Error:\s(.*?)\n\s{4}at\s.*$/s, '$1'));
+        throw new Error(normalizeEvalError(error));
       } else {
         throw error;
       }
     } finally {
       await page?.close();
     }
   }
+
+  async getStorybookVersion(url) {
+    let version, page;
+    let aboutUrl = new URL('?path=/settings/about', url).href;
```

Comment:
Small style thing I wasn't going to comment on until I made the other comments. Can we list uninitialized variables last? (version and page under the url var)